soc/nordic/common/uicr/gen_uicr/Kconfig

```diff
@@ -46,6 +46,91 @@ config GEN_UICR_PROTECTEDMEM_SIZE_BYTES
 	  Size of the protected memory region in bytes.
 	  This value must be divisible by 4096 (4 kiB).
 
+config GEN_UICR_WDTSTART
+	bool "Enable UICR.WDTSTART"
+	help
+	  When enabled, the UICR generator will configure the
+	  watchdog timer to start automatically before the
+	  application core is booted.
```

Comment:
Should we mention that its the local watchdog timer instance as opposed to the global one?